payroll-processor.code-workspace

```diff
@@ -52,7 +58,9 @@
       "EditorConfig.EditorConfig",
       "ms-vscode.vscode-typescript-tslint-plugin",
       "dozerg.tsimportsorter",
-      "stylelint.vscode-stylelint"
+      "stylelint.vscode-stylelint",
+      "k--kato.docomment",
```

Comment:
These are some convenient C#/.NET extensions to help bridge the gap between VS Code and VS